docs/indexing_JSON.md

```diff
@@ -0,0 +1,208 @@
+# Indexing JSON documents
+
+In addition to (JSON) documents store, it is also possible to index JSON documents, gaining full-text search capabilities and document retrieving based on their content. To do so, you must install both modules, RedisJSON and RediSearch.
+
+## Prerequisites
+
+What do you need to start indexing JSON documents?
+
+- Redis 6.x or later
+- RediJSON 2.0 or later
+- RediSearch 2.2 or later
+
+## How to index JSON documents
+
+Let's start by creating an index.
+
+We can now specify `ON JSON` to inform RediSearch that we want to index JSON documents.
+
+Then, on the `SCHEMA` part, you can provide JSONPath expressions.
+The result of each `JSON Path` expression is indexed and associated with a logical name (`attribute`).
+Use the attribute name in the query part.
+
+Here is the basic syntax for indexing a JSON document:
+
+    FT.CREATE {index_name} ON JSON SCHEMA {json_path} AS {attribute} {type}
```

Comment:
Should we emphasize that it is currently **required** to define an alias, in order to use it as an attribute in `FT.SEARCH` and `FT.AGGREGATE`
(there is a limitation in the query parser that might fail with syntax error with complex JSONPath expressions)